apps/test-led-matrix/src/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include "LEDMatrix.h"
+#include "hw_bridge.h"
+
+int main() {
+  printf("####### STARTING #######\r\n");
+
+  LEDMatrix blinky(PA_0, PA_1, PB_0);
+
+  while (true) {
+    blinky.setState(HWBRIDGE::LEDMATRIX::LEDMatrixState::SOLID_RED);
+    ThisThread::sleep_for(5s);
+    blinky.setState(HWBRIDGE::LEDMATRIX::LEDMatrixState::SOLID_RED);
+    ThisThread::sleep_for(5s);
+    blinky.setState(HWBRIDGE::LEDMATRIX::LEDMatrixState::SOLID_RED);
+    ThisThread::sleep_for(5s);
+    blinky.setState(HWBRIDGE::LEDMATRIX::LEDMatrixState::OFF);
```

Comment:
Can you add tests for flashing different colours and different combinations of flash sequences? You can make `setSolidColor` and `setFlashColor` public for this. Just try to make the test app as thorough as possible to test all the transition cases